README.md

```diff
@@ -151,6 +159,17 @@ mvn clean compile exec:java -P postgres -Dexec.args="-b tpcc -c config/postgres/
 
 this is equivalent to the steps above but eliminates the need to first package and then extract the distribution.
 
+### How to configure Database access
+
+The configuration files located in the `/config` folder define the username and password needed to connect to the databases. They can be adapted on a global level by editing the `*_config.xml` files of the corresponding database.
+
+If there is already a built distribution, the config can also be adapted directly in the `/config` folder of the build.
+
+```xml
+<username>admin</username>
+<password>password</password>
```

Comment:
IMO, these are the obvious ones and somewhat self explanatory.

We should probably have additional docs on what the other fields in those configs do and how to adjust them.

For instance, it's possible to run on a select set of queries, or multiple workload phases with a single config, but that isn't super obviously documented IMO.

Reply:
I can see that they look obvious to experienced users but they are needed for a quick-start of benchbase so I'd like to keep them here if possible :)

Do you think it would be beneficial to add a new .README file to the config folder and explain the advanced things there? - Or move everything related to config there and add a quick navigation link in the github README?

Reply:
My point about those two parameters being obvious is that they're self explanatory  _once_ you know where to look, but the part about where the configs are may not be.

I would suggest that the Getting Started text get rewritten as something slightly more high-level like "adjusting the connection details in the config file" which includes basic commentary about both the JDBC URL and the credentials.

A separate `ConfigDetails.md` might be warranted that the main `README.md` links to for additional details, especially with external links to JDBC driver details.

I don't think the entire `ConfigDetails.md` is necessary for you to write here, but if you could start a stub of it with the above, then I think one of the benchbase experts can take up the rest from there.

Make sense?